packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -692,36 +692,43 @@ export class MergeTree {
             // Will replace this block with a packed block
             childBlock.parent = undefined;
         }
-        const totalNodeCount = holdNodes.length;
-        const halfCount = MaxNodesInBlock / 2;
-        let childCount = Math.min(MaxNodesInBlock - 1, Math.floor(totalNodeCount / halfCount));
-        if (childCount < 1) {
-            childCount = 1;
-        }
-        const baseCount = Math.floor(totalNodeCount / childCount);
-        let extraCount = totalNodeCount % childCount;
-        const packedBlocks = new Array<IMergeBlock>(MaxNodesInBlock);
-        let readCount = 0;
-        for (let nodeIndex = 0; nodeIndex < childCount; nodeIndex++) {
-            let nodeCount = baseCount;
-            if (extraCount > 0) {
-                nodeCount++;
-                extraCount--;
-            }
-            const packedBlock = this.makeBlock(nodeCount);
-            for (let packedNodeIndex = 0; packedNodeIndex < nodeCount; packedNodeIndex++) {
-                const nodeToPack = holdNodes[readCount++];
-                packedBlock.assignChild(nodeToPack, packedNodeIndex, false);
-            }
-            packedBlock.parent = parent;
-            packedBlocks[nodeIndex] = packedBlock;
-            this.nodeUpdateLengthNewStructure(packedBlock);
-        }
-        parent.children = packedBlocks;
-        for (let j = 0; j < childCount; j++) {
-            parent.assignChild(packedBlocks[j], j, false);
-        }
-        parent.childCount = childCount;
+        if (holdNodes.length > 0) {
```

Comment:
i would like to see tests that target the changes here. might make sense to put the zamboni changes in a seperate pr with tests

Reply:
should any new tests for packParent go in the separate zamboni PR as well?

Reply:
since these two causes ended up being independent, i think it's better to just remove these changes from the PR and include them in the zamboni refactor PR (or post-refactor). Usually better to go minimal changes. There's also a stray console.log :P